relay-general/src/store/normalize/span/tag_extraction.rs

```diff
@@ -11,7 +11,15 @@ use crate::protocol::{Event, Span, TraceContext};
 use crate::store::utils::{
     extract_http_status_code, extract_transaction_op, get_eventuser_tag, http_status_code_from_span,
 };
-use crate::types::Annotated;
+use crate::types::{Annotated, Value};
+
+/// Used to decide when to extract mobile-specific span tags.
+const MOBILE_SDKS: [&str; 4] = [
+    "sentry.cocoa",
+    "sentry.dart.flutter",
+    "sentry.java.android",
+    "sentry.javascript.react-native",
```

Comment:
@shruthilayaj Not sure if this list is complete.

Reply:
yep, this looks good to me! these are the ones we plan to support in the first pass